markdown.go

```diff
@@ -312,29 +320,21 @@ func MarkdownBasic(input []byte) []byte {
 	return Markdown(input, renderer, Options{})
 }
 
-// Call Markdown with most useful extensions enabled
-// MarkdownCommon is a convenience function for simple rendering.
-// It processes markdown input with common extensions enabled, including:
+// MarkdownCommon is a convenience function for simple rendering. It calls
+// Markdown with most useful extensions enabled, including:
 //
 // * Smartypants processing with smart fractions and LaTeX dashes
-//
 // * Intra-word emphasis suppression
-//
 // * Tables
-//
 // * Fenced code blocks
-//
 // * Autolinking
-//
 // * Strikethrough support
-//
 // * Strict header parsing
-//
```

Comment:
I think these blank lines are needed to keep the bullet points as separate paragraphs.

https://godoc.org/gopkg.in/russross/blackfriday.v2#MarkdownCommon

Otherwise, all of them will be merged into a single paragraph, which I'm sure is not what you intended:

![image](https://cloud.githubusercontent.com/assets/1924134/17200141/b9400f24-5450-11e6-8a39-e855da0b0f3b.png)